pkg/security/ldapclient/interfaces.go

```diff
@@ -1,6 +1,6 @@
 package ldapclient
 
-import "gopkg.in/ldap.v2"
```

Comment:
this is not part of a bump. A bump are just the vendor changes, and potentially the glide/go.mod changes. Move this into a fix-up commit.